### ISSUE TITLE
Use with monorepos

### ISSUE THREAD

**Issue by snoblenet** [natecraddock/workspaces.nvim#40]
I have:

```
/traditional_repo
  file_1
/mono_repo
  file_2
  /packages
    /package
      file_3
```

Is it practical to configure `workspaces.nvim` so that:

- `file_1` belongs to `traditional_repo`, 
- `file_2` belongs to `mono_repo`, and
- `file_3` belongs to `package`?

**Comment by natecraddock** [comment]
You could register `traditional_repo` `mono_repo` and `package` as separate workspaces with this plugin just fine. I'm not sure if that is what you are asking about though!

**Comment by snoblenet** [comment]
It is. But related to [my other issue](https://github.com/natecraddock/workspaces.nvim/issues/39), this is something I'd want to control manually.

Also, by default, would `file_3` then belong to two workspaces? (`mono_repo` and `package`)

**Comment by natecraddock** [comment]
Yes file_3 would be in two separate workspaces. All a workspace does is allow you to set the working directory of nvim
